README.md

```diff
@@ -130,20 +139,88 @@ sudo apt-get install -y libavcodec-dev libavformat-dev libswscale-dev libv4l-dev
 sudo apt-get install -y libatlas-base-dev gfortran
 ```
 
-### 2. Install project dependencies
+#### 2. Clone the repo
+
+```bash
+git clone https://github.com/Udayraj123/OMRChecker
+cd OMRChecker/
+```
+
+#### 3. Install pip requirements
+
+```bash
+python3 -m pip install --user -r requirements.txt
+```
+
+**Friendly Note:** If you face a distutils error in pip, use `--ignore-installed` flag in above command.
+
+<!-- Wiki should not get cloned -->
+</details>
+
+<details>
+<summary><b><u>Windows Users</u></b></summary>
+
+**Note:** all given commands are in bash
+
+#### 1. Install global dependencies
+
+![opencv 4.0.0](https://img.shields.io/badge/opencv-4.0.0-blue.svg) ![python 3.4+](https://img.shields.io/badge/python-3.4+-blue.svg)
+
+Install opencv (Any installation method is fine.)
+
+```bash
+python3 -m pip install --user --upgrade pip
+python3 -m pip install --user opencv-python
+python3 -m pip install --user opencv-contrib-python
+```
+
+More details on pip install openCV [here](https://www.pyimagesearch.com/2018/09/19/pip-install-opencv/).
+
+#### 2. Clone the repo
```

Comment:
From 2. Clone the repo. The part will stay exactly the same for all operating systems. Let's not repeat it for each, and keep it below all the folded parts. 
<img width="691" alt="image" src="https://user-images.githubusercontent.com/16881051/193468834-93f30798-8a3f-4371-b778-5c002a2b14ea.png">